docs/en/Overview.md

```diff
@@ -1,142 +1,49 @@
-Other scenarios also reap benefits as well; 9x throughput improvement can be achieved when sequentially reading an unstructured file.
-
-![Dora read approaches]({{ '/img/dora_read_approaches.png' | relativize_url }})
-
-### Decentralized Metadata Store
-
-DORA spreads metadata to every worker to ensure that metadata is always accessible and available.
-To optimize metadata access, DORA utilizes a two-level caching system for metadata entries.
-The first level of caching is the in-memory cache, which stores metadata entries in memory.
-This cache has a configurable maximum capacity and time-to-live (TTL) setting to set an expiration duration.
-The second level of caching is the persistent cache, which stores metadata entries on disk using RocksDB.
-The persistent cache has unbounded capacity, depending on available disk space,
-and also uses TTL-based cache eviction, avoiding any active sync or invalidation.
-
-The combination of in-memory and persistent caching helps ensure that metadata is readily available and accessible,
-while also allowing for efficient use of system resources.
-The decentralization of metadata avoids the bottleneck in the architecture where metadata is primarily managed by the worker nodes.
-With the capability of storing 30 to 50 million files per worker,
-the system can support large-scale data-intensive applications with billions of files.
-
-### Zero-copy Networking
-
-DORA provides a Netty-based data transmission solution that offers a 30%-50% performance improvement over gRPC.
-This solution has several advantages, including fewer data copies through different thread pools,
-zero-copy transmission that avoids serialization of Protobuf, optimized off-heap memory usage that prevents memory capacity issues,
-and less data transfer due to the absence of additional HTTP headers.
+## Get Started with Alluxio
 
-![Zero copy network]({{ '/img/zero_copy_network.png' | relativize_url }})
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5YQvvznT5cI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-### Scheduler and Distribute Load
+Check out our [Quick Start Guide](/en/Get-Started.html) for step-by-step guidance on how to download Alluxio, run Alluxio on a local machine, and perform basic tasks via Alluxio Shell.
 
-The scheduler provides an intuitive, extensible solution for efficient job scheduling,
-with consideration towards observability, scalability, and reliability.
-It has also been used to implement a distributed load capable of loading billions of files.
+Follow along to deploy an Alluxio cluster with High Availability (HA) by running multiple Alluxio master processes on different nodes in the system. [Read More](/en/deploy/Deploy-Alluxio-on-Cluster-with-HA.html)
+<br />
+<br />
 
-## Benchmark Results
+## Compute Integrations
```

Comment:
i don't want to list the nav bar pages again in a page. for example, people are not going to remember to update this section when adding a new compute integration.